project/AppDependencies.scala

```diff
@@ -18,13 +17,13 @@ object AppDependencies {
   val test: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"           %% "bootstrap-test-play-28"   % bootstrapVersion,
     "uk.gov.hmrc.mongo"     %% "hmrc-mongo-test-play-28"  % mongoVersion,
-  "org.scalatestplus"       %% "scalacheck-1-17"          % "3.2.16.0",
-  "org.scalatest"           %% "scalatest"                % "3.2.16",
-  "org.jsoup"               %  "jsoup"                    % "1.16.1",
-  "org.mockito"             %% "mockito-scala-scalatest"  % "1.17.14",
-  "com.github.tomakehurst"  %  "wiremock-standalone"      % "2.27.2",
-  "wolfendale"              %% "scalacheck-gen-regexp"    % "0.1.2",
-  "com.vladsch.flexmark"    %  "flexmark-all"             % "0.64.8"
+    "org.scalatestplus"     %% "scalacheck-1-17"          % "3.2.16.0",
+    "org.scalatest"         %% "scalatest"                % "3.2.16",
+    "org.jsoup"             %  "jsoup"                    % "1.16.1",
+    "org.mockito"           %% "mockito-scala-scalatest"  % "1.17.22",
+    "org.wiremock"          %  "wiremock-standalone"      % "3.0.1",
```

Comment:
I don't think there's a version `3.0.1` for this dependency. Using `3.0.0` should be fine though

Reply:
version `3.0.1` is fine - released yesterday